lib/rpmsg/rpmsg_virtio.c

```diff
@@ -492,7 +492,14 @@ static int rpmsg_virtio_ns_callback(struct rpmsg_endpoint *ept, void *data,
 		} else {
 			_ept->dest_addr = dest;
 			metal_mutex_release(&rdev->lock);
+			if (rdev->support_ack)
+				rpmsg_send_ns_message(_ept, RPMSG_NS_CREATE_ACK);
```

Comment:
The ack looks like a "start" message to me. it is better to send after the local endpoint is created, and send it through the new created local endpoint. The "ack" indicates the one site is ready to receive

Reply:
Yes, RPMSG_NS_CREATE_ACK is unnecessary and confused. I will remove it.